jscomp/all.depend

```diff
@@ -946,9 +946,9 @@ bsb/bsb_default.cmi : bsb/bsb_config_types.cmx
 bsb/bsb_dir_index.cmi :
 bsb/bsb_exception.cmi : ext/ext_position.cmi ext/ext_json_types.cmx
 bsb/bsb_file.cmi :
+bsb/bsb_helper_depfile_gen.cmi : bsb/bsb_dir_index.cmi
 bsb/bsb_helper_dep_graph.cmi : ext/string_set.cmi ext/string_map.cmi \
     ext/string_hashtbl.cmi
-bsb/bsb_helper_depfile_gen.cmi : bsb/bsb_dir_index.cmi
 bsb/bsb_helper_extract.cmi : ext/string_set.cmi
```

Comment:
can you not check in such changes, seems not relevant here

Reply:
@bobzhang Sorry, it's been reverted.